app/models/test_passage.rb

```diff
@@ -19,20 +23,24 @@ def accept!(answer_ids)
     save!
   end
 
-  def completed?
-    current_question.nil?
-  end
-
   def test_passed?
     (correct_questions / test.questions.count * 100).floor >= PASS_TEST_THRESHOLD
-  end
+  end  
 
   def current_question_number
     test.questions.where('id < ?', current_question.id).count + 1
   end
 
+  def values_for_result
```

Comment:
```suggestion
  def result_percent
```
Этот метод стоит сделать приватным и использовать [тут](https://github.com/dastanabeuov/TestGuru/pull/21/files#diff-55b98c5a489d9b6a093953a2765478deR27)

Reply:
https://github.com/dastanabeuov/TestGuru/pull/21/files#r256695448